R/dvhydrograph-render.R

```diff
@@ -89,6 +97,11 @@ createDVHydrographPlot <- function(reportObject){
   estimated3EdgesDf <- as.data.frame(estimated3Edges, stringsAsFactors = FALSE)
   estimated4EdgesDf <- as.data.frame(estimated4Edges, stringsAsFactors = FALSE)
   comparisonEdgesDf <- as.data.frame(comparisonEdges, stringsAsFactors = FALSE)
+  
+  #remove zeros from field measurements, if present:
+  if (excludeZeroNegativeFlag && !isEmptyOrBlank(fieldVisitMeasurements)) {
+    fieldVisitMeasurements <- removeZeroNegative(fieldVisitMeasurements)
+  }
```

Comment:
Right, but `isLogged` will only return true if it won't cause an error, and that function checks `excludeZeroNegativeFlag` inside. So this `if` can just become 

```
if (logAxis) {
    fieldVisitMeasurements <- removeZeroNegative(fieldVisitMeasurements)
}
```